packages/host/tests/acceptance/ai-assistant-test.gts

```diff
@@ -60,18 +61,22 @@ import {
   field,
   setupBaseRealm,
   StringField,
-  SystemCard,
-  ModelConfiguration,
 } from '../helpers/base-realm';
 
 import { setupMockMatrix } from '../helpers/mock-matrix';
 import { getRoomIdForRealmAndUser } from '../helpers/mock-matrix/_utils';
 import { setupApplicationTest } from '../helpers/setup';
 
+const catalogRealmURL = ensureTrailingSlash(ENV.resolvedCatalogRealmURL);
+const skillsRealmURL = ensureTrailingSlash(ENV.resolvedSkillsRealmURL);
```

Comment:
Could this be elevated to happen in `config/environment` so it doesn’t have to happen everywhere config values are consumed?